pkg/controller/node/node_controller.go

```diff
@@ -554,22 +554,25 @@ func getCandidateMachines(pool *mcfgv1.MachineConfigPool, nodesInPool []*corev1.
 }
 
 func maxUnavailable(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) (int, error) {
-	// Hardcoded for now per discussion in
-	// https://github.com/openshift/machine-config-operator/pull/701
-	if pool.Name == "master" {
-		return 1, nil
-	}
 	intOrPercent := intstrutil.FromInt(1)
```

Comment:
random Q but why are we using IntOrString types?  bc people might say: 1 (node) or 20%?
Like I'm a little confused by it.  Maybe we could add some comments?

Reply:
I see that I was right by looking at the test, but I think a 1 line here would be good for when people skim later. :smile:

Reply:
> Maybe we could add some comments?

Code is pretty explicit, maybe we should add some examples around that in docs/ or examples/